.npmignore

```diff
@@ -0,0 +1,12 @@
+yarn-error.log
+.sass-cache/
+.cache/
+jscov/
+.nyc_output/
+test/
+*~
+.livereload
+*.orig
+*.css.map
+*.js.map
+.circleci/
```

Comment:
Normally, an npm package uses `.gitignore` to decide what to include in the packaged version. Since we're now git-ignoring `dist/`, which is required by Herman to work, I added `.npmignore` so that we can define them independently.